scripts/doc-checker.py

```diff
@@ -22,7 +22,19 @@
 
 
 # code snippet from numpydoc
-def validate_object(import_path):
+def validate_object(import_path: str) -> int:
```

Comment:
I suggest changing return type to `bool` (this includes returning `True` and `False`, of course). I think original code gave either ratings or at least `OK`/`Warnig`/`Error` while we just need `fine` vs `not fine` outcome.